pkg/openevec/edenConfig.go

```diff
@@ -124,10 +124,10 @@ func ConfigAdd(cfg *EdenSetupArgs, currentContext, contextFile string, force boo
 		imageDist := fmt.Sprintf("%s-%s", context.Current, defaults.DefaultImageDist)
 		switch cfg.Eve.Arch {
 		case "amd64":
-			viper.Set("eve.firmware", []string{filepath.Join(imageDist, "eve", "OVMF_CODE.fd"),
-				filepath.Join(imageDist, "eve", "OVMF_VARS.fd")})
+			viper.Set("eve.firmware", []string{filepath.Join(imageDist, "eve", "firmware", "OVMF_CODE.fd"),
```

Comment:
Curious, something changed in image or it is because of another extract approach?

Reply:
It's because in the new approach we extract the whole directory instead of the contents of the directory. It's just a question of style, but I think it better shows the separation between the OVMF* files and the img files that go into `dist/default-images/eve`.